pages/translations/en/LC_MESSAGES/messages.po

```diff
@@ -1147,7 +1150,1930 @@ msgstr ""
+msgstr ""
+
+msgid "WYSIWYG drag and drop editor with templates, illustrations, icons, and the ability to upload your own images and videos."
+msgstr ""
+
+msgid "Newsroom Studio helps you create engaging, instant loading mobile experiences."
+msgstr ""
+
+msgid "Unbounce is a platform to create landing pages, website popups, and sticky bars."
+msgstr ""
+
+msgid "AMP-first content creation with the power and flexibility of WordPress."
+msgstr ""
+
+msgid "Bold by Quintype is a headless CMS built for enterprise publishers with built-in support for AMP."
+msgstr ""
+
+msgid "Helps you measure your site's Search traffic and performance, fix issues, and make your site shine in Google Search results."
+msgstr ""
+
+msgid "Test if and how your AMP page appears in Google Search."
+msgstr ""
+
+msgid "Platform to share and discover how-to stories that includes a creation tool on your phone to help you create how-to stories."
+msgstr ""
+
+msgid "Build AMP sites, posts, and stories. Win search, social, and email. Fast. Flexible. Secure. Modern."
+msgstr ""
+
+msgid "Platform to share and discover how-to stories that includes a creation tool on your phone to help you create how-to stories."
+msgstr ""
+
+msgid "Essential component for AMP development and available in a variety of versions."
+msgstr ""
+
+msgid "Beta-test new features of AMP before they're available for everyone."
+msgstr ""
+
+msgid "Helps you find out whether your Google Analytics and AdWords tracking is correctly set up."
+msgstr ""
+
+msgid "A collection of AMP tools making it easier to publish and host AMP pages."
+msgstr ""
+
+msgid "Validation tool that offers additional debug information on top of the AMP validator."
+msgstr ""
+
+msgid "Pre-built Google Analytics dashboard template that helps you get to your analytics insights faster."
+msgstr ""
+
+msgid "Chrome extension that helps you understand the feasibility of converting your site to AMP."
+msgstr ""
+
+msgid "Generate custom templates for your AMP websites, stories, emails or ads."
+msgstr ""
+
+msgid "Start prototyping in an interactive online editor with the ability to use templates and our examples."
+msgstr ""
+
+msgid "A prototyping tool that automatically converts a HTML page to AMP for demonstrating performance gains with AMP."
+msgstr ""
+
+msgid "Litmus makes it easy to build, test, and monitor every email for the best results."
+msgstr ""
+
+msgid "Makes authoring AMP easy by validating for errors/warnings as you type within VS Code."
+msgstr ""
+
+msgid "React framework that generates AMP and PWA pages from a single codebase."
+msgstr ""
+
+msgid "A tool to help for prototyping ads to be served in AMP Stories. Choose from pre-existing templates to get started."
+msgstr ""
+
+msgid "NodemailerApp is an email debugging app that includes a local SMTP server and renders AMP for Email emails."
+msgstr ""
+
+msg
```

Comment:
product name

Reply:
I don't think so?

![image](https://user-images.githubusercontent.com/465414/74041674-f2689780-497a-11ea-84a1-9cedabe32cac.png)

the strings are separated because that the page is rendered in two different elements

Reply:
I don't understand the comment in relation to `RebelMouse`

Reply:
hm, I feel like there was some kind of miscommunication. The line referenced was the line I highlighted in the image. It might have changed from an update on my end.

Are you saying to remove this?

```
msgid "RebelMouse"
```